consensus/ethash/ethash_test.go

```diff
@@ -27,16 +27,15 @@ import (
 	"testing"
 	"time"
 
+	"golang.org/x/exp/slog"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/log"
 )
 
 func verboseLogging() {
-	glogger := log.NewGlogHandler(log.StreamHandler(os.Stdout, log.TerminalFormat(false)))
-	glogger.Verbosity(log.Lvl(99))
-	log.Root().SetHandler(glogger)
+	slog.SetDefault(slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug})))
```

Comment:
Why we don't use the `go-ethereum/log` here?

Reply:
"golang.org/x/exp/slog" is used in the log package either way. But can't we use that instead and keep the dependency just there?